docs/setup.md

```diff
@@ -2,45 +2,50 @@
 
 This document outlines the setup process for the `Superposition Platform`.
 
-## Prerequisites
+## Installation
 
-* Docker installed and running.
-* Docker Compose installed.
-* [Follow installation instructions at Zero To Nix ↗](https://zero-to-nix.com/start/install) 
+- Install Rust using rustup
+    `curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh`
+- Install wasm-pack
+    `curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh`
+- Install docker
+- make sure you have dependencies like libpq, openssl, libiconv,etc installed
+- Clone the repository 
+    ```bash
+    git clone https://github.com/juspay/superposition.git
+    cd superposition
+    ```
+- Start docker daemon
+    ```bash
+    open --background -a Docker
```

Comment:
This is MacOS specific I believe (should we give the Linux alternative as well)

Reply:
Yeah let me add that